.circleci/config.yml

```
@@ -37,6 +37,7 @@ workflows:
     jobs:
       - test
       - release:
+          context: frontend-deploy-library
```

This looks like the context we use for private CircleCI (see [pay-in](https://github.com/transferwise/pay-in/blob/4fa66eb76ea5498015f2ade410d8a38c7aa74c83/.circleci/config.yml#L101)), public repos have this (see [public-navigation](https://github.com/transferwise/public-navigation/blob/c1a732d9995798a679c3806978a1343a44780beb/.circleci/config.yml#L132)):
```suggestion
          context: frontend-publish
```

---

this is a dangerous naming convention